test/images/scripts/debian.setup

```diff
@@ -0,0 +1,156 @@
+#! /bin/bash
+# Shared .setup between all Debian/Ubuntu flavors
+
+set -ex
+
+# Enable a console on ttyS0 so that we can log-in via vm-run.
+# and make the boot up more verbose
+sed -i 's/GRUB_CMDLINE_LINUX_DEFAULT/# GRUB_CMDLINE_LINUX_DEFAULT/' /etc/default/grub
+echo 'GRUB_CMDLINE_LINUX="console=ttyS0,115200 net.ifnames=0 biosdevname=0"' >>/etc/default/grub
+update-grub
+
+# We install all dependencies of the cockpit packages since we want
+# them to not spontaneously change from one test run to the next when
+# the distribution repository is updated.
+# docker.io \
+#
+COCKPIT_DEPS="\
+cryptsetup \
+docker.io
+libjson-glib-1.0-0 \
+libpcp3 \
+libpolkit-agent-1-0 \
+libpolkit-gobject-1-0 \
+libpwquality-tools \
+libssh-4 \
+libteam-utils \
+libvirt-daemon-system \
+network-manager \
+pcp \
+policykit-1 \
+realmd \
+selinux-basics \
+thin-provisioning-tools \
+xdg-utils \
+udisks2 \
+"
+
+# We also install the packages necessary to join a FreeIPA domain so
+# that we don't have to go to the network during a test run.
+IPA_CLIENT_PACKAGES="\
+sssd-tools \
+packagekit \
+"
+
+TEST_PACKAGES="\
+curl \
+gdb \
+iproute \
+mdadm \
+qemu-kvm \
+virtinst \
+xfsprogs \
+"
+
+RELEASE=$(grep -m1 ^deb /etc/apt/sources.list | awk '{print $3}')
+case "$RELEASE" in
+    stable)
+        # docker.io is in backports only
+        echo "deb http://deb.debian.org/debian jessie-backports main" >/etc/apt/sources.list.d/jessie-backports.list
```

Comment:
Using jessie-backports with debian-9 is probably wrong, no?  Can we protect against this and make this fail once Debian 9 becomes `stable`?  (Or somehow make it so that $RELEASE is "jessie" instead of "stable".)